pages/dkp/kommander/2.2/licensing/essential/index.md

```diff
@@ -0,0 +1,63 @@
+
+## Platform applications
+
+When creating a cluster, the application manager deploys certain platform applications on the newly created cluster. DKP Essential users can use the Kommander UI to customize which platform applications to deploy to the cluster in a given workspace. For a list of available platform applications that are included with DKP Essential, see [Workspace Platform Applications][workspaceplatform].
+
+<p class="message--note"><strong>NOTE:</strong> The platform application <code>kubecost</code> is not included with DKP Essential, but is included with <a href="/dkp/kommander/2.2/licensing/enterprise/">DKP Enterprise</a>.</p>
+
+## Cluster manager
+
+Konvoy is the Kubernetes installer component of DKP Essential that uses industry standard tools to produce a certified Kubernetes cluster. These industry standard tools create a cluster management system that includes:
+
+- **Control Plane:** Manages the worker nodes and pods in the cluster.
+- **Worker Nodes:** Used to run containerized applications and handle networking to ensure that traffic between applications across the cluster and from outside of the cluster can be properly facilitated.
+- **Container Networking Interface (CNI):** Calico's open source networking and network security solution for containers, virtual machines, and native host-based workloads.
+- **Container Storage Interface (CSI):** A common abstraction to container orchestrators for interacting with storage subsystems of various types.
+- **Kubernetes Cluster API (CAPI):** Cluster API uses Kubernetes-style APIs and patterns to automate cluster lifecycle management for platform operators. For more on how CAPI is integrated into DKP Essential, see [Understanding CAPI Concepts and Terms][capi-concepts-and-terms]
+- **Cert Manager:** A Kubernetes addon to automate the management and issuance of TLS certificates from various issuing sources.
+- **Cluster Autoscaler:** A component that automatically adjusts the size of a Kubernetes cluster so that all pods have a place to run and there are no unneeded nodes.
+
+## Built-in GitOps
+
+DKP Essential comes bundled with GitOps, an operating model for Kubernetes and other cloud native technologies, providing a set of best practices that unify Git deployment, management and monitoring for containerized clusters and applications. GitOps works by using Git as a single source of truth for declarative infrastructure and applications. With GitOps, the use of software agents can alert on any divergence between Git with what is running in a cluster, and if there's a difference, Kubernetes reconcilers automatically update or rollback the cluster depending on the case.
+
+## DKP Essential single cluster UI
+
+Bundled with DKP Essential is a single cluster management UI that can be used in lieu of the bundled CLI. From the UI you can:
```

Comment:
Let's make it: 
Bundled with DKP Essential is a single cluster management view of DKP UI that can be used in lieu of the bundled CLI. From the UI you can: